docs/build-aea-programmatically.md

```diff
@@ -71,6 +73,40 @@ Next, we add the echo skill which will bounce our messages back to us. We first
     builder.add_skill("./packages/fetchai/skills/echo")
 ```
 
+Also, we can add a component that was instantiated programmatically. :
+``` python
+    # create skill and handler manually
+    from aea.protocols.base import Message
+    from aea.protocols.default.message import DefaultMessage
+    from aea.skills.base import Handler
+
+    class DummyHandler(Handler):
+        """Dummy handler to handle messages."""
+
+        SUPPORTED_PROTOCOL = DefaultMessage.protocol_id
+
+        def setup(self) -> None:
+            """Noop setup."""
+
+        def teardown(self) -> None:
+            """Noop teardown."""
+
+        def handle(self, message: Message) -> None:
+            """Handle incoming message."""
+            print("Hi, you got a ", message)
```

Comment:
please replace with `self.context.logger.info("You got a message: {}".format(str(message)))`

Reply:
done, but suppose better to use f-strings